tests/phpunit/Utils/FunctionsTest.php

```diff
@@ -85,4 +85,64 @@ public function testInvalidUrlPaths(string $input, bool $expected): void
         $url = isValidUrlPath($input);
         $this->assertEquals($url, $expected);
     }
+
+
+    //=======================
+    //=======================
+    //=======================
+
+    public static function isValidPathProvider(): array
+    {
+        return [
+            'Accept valid path'   => [
+                '/ko-kr/range/%EB%B0%9C%EB%A0%8C%ED%83%80%EC%9D%B87%EB%85%84-%EB%B2%84%EB%B2%88%ED%94%BC%EB%8B%88%EC%89%AC/',
+                true
+            ],
+            'Must start with /'    => [
+                '/hello/world/',
+                true
+            ],
+            'Must start with / [2]'    => [
+                'hello/world/',
+                false
+            ],
+            'Accept non latin paths'    => [
+                '/zh-cn/条款和条件/',
+                true
+            ],
+            'Reject non latin paths [2]'    => [
+                '/zh-cn/إسرائيل/',
+                true
+            ],
+            'Reject invalid url path'    => [
+                '/zh-cn/<hello></hello>',
+                false
+            ],
+            'Reject path with non ascii characters'    => [
+                '/en/AC/Ææ½⅓¼⅕⅙⅐⅛⅑/',
+                true
+            ],
+            'Reject path with non ascii characters [2]'    => [
```

Comment:
do we accept this ?

Reply:
yes, that is what the ticket says but also some of our sites do have those non ascii chars in the url. Like the chinese characters (see Chivas , RoyalSalute)